sycl/source/detail/scheduler/commands.cpp

```diff
@@ -681,15 +681,19 @@ static void adjustNDRangePerKernel(NDRDescT &NDR, RT::PiKernel Kernel,
 
   if (WGSize[0] == 0) {
     // kernel does not request specific workgroup shape - set one
-    // TODO maximum work group size as the local size might not be the best
-    //      choice for CPU or FPGA devices
-    size_t WGSize1D = get_kernel_work_group_info<
-        size_t,
-        cl::sycl::info::kernel_work_group::work_group_size>::get(Kernel,
-                                                                 Device);
-    assert(WGSize1D != 0);
-    // TODO implement better default for 2D/3D case:
-    WGSize = {WGSize1D, 1, 1};
+    id<3> MaxWGSizes = get_device_info<
+        id<3>, cl::sycl::info::device::max_work_item_sizes>::get(Device);
+
+   size_t WGSize1D = get_kernel_work_group_info<
```

Comment:
It seems formatting is broken here. Please fix.

Reply:
Well spotted, I will fix this.

Reply:
Fixed.